lua/kulala/parser/curl.lua

```diff
@@ -32,7 +32,10 @@ function M.parse(curl)
 
   for _, arg in ipairs(parts) do
     if state == State.START then
-      if arg == "-X" or arg == "--request" then
+      if arg:match("^http[s]?://") and res.url == "" then
```

Comment:
Wouldn't it be better to make it match `://` so we are maybe more future-proof?

Reply:
https://github.com/mistweaverco/kulala.nvim/pull/186/commits/7f31918e127d0b85a292db9c8c3b44f3b80b3a50